resources/views/grid.blade.php

```diff
@@ -0,0 +1,28 @@
+@extends('components.content-area')
+
+@section('content')
+    @include('components.page-title', ['title' => $page['title']])
+
+    <div class="content">
+        {!! $page['content']['main'] !!}
+    </div>
+
+    @if(!empty($grid))
+        <div class="flex flex-wrap -mx-4">
```

Comment:
How is this block different than `components/promo-grid.blade.php`?

Reply:
This was a duplicate, thanks for catching